Makefile

```diff
@@ -18,6 +18,7 @@ all: build
 SOURCEDIR=./ecr-login
 SOURCES := $(shell find $(SOURCEDIR) -name '*.go')
 VERSION := $(shell cat VERSION)
+GITFILES := $(shell find ".git/")
 GITCOMMIT_SHA := $(shell git rev-parse --short HEAD)
```

Comment:
Is there any reason to or not to eliminate this variable? On the surface, this look like it could be replaced in many usages with a dependency on the GITCOMMIT_SHA file and `$(shell cat GITCOMMIT_SHA)`.

Not a blocker, but I'm curious.

Reply:
Oh, I did mean to remove the GITCOMMIT_SHA variable.  Will do that.